playbook/app/pb_kits/playbook/index.js

```diff
@@ -2,7 +2,7 @@ import './_playbook.scss'
 import 'lazysizes/plugins/attrchange/ls.attrchange'
 import 'lazysizes'
 
-// React Component JSX Imports from the React Kits
+// vvv React Component JSX Imports from the React Kits vvv
```

Comment:
Might be worth leaving another comment here saying `// The comment below is used by the kit generator to alphabetize imports, moving or removing it will break functionality` or something to that effect.  

Also might be good to name it more explicitly, something like `start-flag` and `end-flag`.  I like the concept though, clever solution.

Reply:
I was just airing on the side of caution by keeping the same wording but adding arrows. During dailies today we can see what the rest of the team would think is a good idea. Something like an extra line above and below the comments like DO NOT REMOVE BELOW/ ABOVE?

As for adding a method for the kits that seems easy enough 👷🏽‍♀️

Reply:
Generally speaking, it's probably a good idea to just add something to the effect of 

> This file is autogenerated. Please do not edit.